specifyweb/frontend/js_src/lib/utils/parser/definitions.ts

```diff
@@ -298,8 +298,11 @@ export function mergeParsers(base: Parser, extra: Parser): Parser {
         'required',
         base?.required === true || extra?.required === true ? true : undefined,
       ],
-      ...concat
-        .map((key) => [key, [...(base[key] ?? []), ...(extra[key] ?? [])]])
+      ...uniqueConcat
```

Comment:
fix bug when `base` and `extra` are the same parser, resulting in formatters being concatenated, thus applied twice, thus crashing